### DISCUSSION TITLE
MonitorControl affecting screen sharing?

### DISCUSSION THREAD

**Discussion by tallavi** [MonitorControl/MonitorControl#1380] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Thanks for the great app!
I don't know if it counts as a bug, but the overlay is not only seen by me, but the screen is darkened for people who I share my screen with in zoom meetings and slack huddles.

Is there any chance to hide this overlay from them, and to it only be visible for my eyes only?

Thanks!

### Steps to reproduce

1. Darken the screen using MonitorControl
2. Start a zoom meeting
3. On the other side, the screen is also darkened.

### Expected behavior

On the other side, the screen should be with it's natural brightness.

### Environment Information (please complete the following information)

```markdown
- macOS version: 13.4 (22F66)
- Mac model: 16-inch, 2019
- MonitorControl version: 4.1.0 (7034)
- Monitor(s): internal
- Apple Silicon/M1 (yes or no): no
```

**Comment by waydabber** [comment]
MC should not affect screensharing unless gamma table manipulation is disabled (intentionally or because of a virtual display like SideCar or DisplayLink) and the app falls back to overlay dimming. In this case this is expected.

**Reply by tallavi** [reply]
Yep, that was the problem. I guess I disabled gamma manipulation while testing and setting up (I thought I had a incompatibility issue with my monitor). Looks good now. Thanks!
